hack/verify-crds.sh

```
@@ -8,11 +8,11 @@ for f in $HUB_CRD_FILES
 do
     if [ -f "$BASE_DIR/$(basename $f).yaml-patch" ];
     then
-        $1 -o $BASE_DIR/$(basename $f).yaml-patch < $f > ./manifests/cluster-manager/hub/$(basename $f).tmp
-        diff -N ./manifests/cluster-manager/hub/$(basename $f).tmp ./manifests/cluster-manager/hub/$(basename $f) || ( echo 'crd content is incorrect' && false )
-        rm ./manifests/cluster-manager/hub/$(basename $f).tmp
+        $1 -o $BASE_DIR/$(basename $f).yaml-patch < $f > ./manifests/cluster-manager/hub/crds/$(basename $f).tmp
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)